### DISCUSSION TITLE
是否有支援 Polkadot API

### DISCUSSION THREAD

**Discussion by iisaint** [consenlabs/webview#33] [Q&A] [answered]
我想透過 imtoken 來跟Polkadot上面的 Dapp做互動，api文件沒找到相關內容，請問是否有支援？

**Comment by unix** [comment]
@xwartz

**Comment by xwartz** [comment, marked as answer]
imToken 有拦截 polkadot.js 的方法，直接按照 [polkadot.js 文档](https://polkadot.js.org/docs/) 就行
